Sources/PubNub/PubNub.swift

```diff
@@ -985,7 +1001,7 @@ public extension PubNub {
     for channels: [String],
     includeActions: Bool = false, includeMeta: Bool = false,
     includeUUID: Bool = true, includeMessageType: Bool = true,
-    page: PubNubBoundedPage? = PubNubBoundedPageBase(),
+    includeSpaceId: Bool = false, page: PubNubBoundedPage? = PubNubBoundedPageBase(),
```

Comment:
Nitpick: You can move the `page` parameter to the new line

Reply:
@jguz-pubnub I prefer to have one parameter per-line, if they all doesn't fit into single line. But In this file it looks like two parameters at a line, so I tried to keep with file style.